website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1109 @@ update your API calls accordingly.
+- [Issuing Certificates](#issuing-certificates)
+  - [List Roles](#list-roles)
+  - [Generate Certificate and Key](#generate-certificate-and-key)
+  - [Sign Certificate](#sign-certificate)
+  - [Sign Intermediate](#sign-intermediate)
+  - [Sign Self-Issued](#sign-self-issued)
+  - [Sign Verbatim](#sign-verbatim)
+  - [Revoke Certificate](#revoke-certificate)
+- [Managing Keys and Issuers](#managing-keys-and-issuers)
+  - [List Issuers](#list-issuers)
+  - [List Keys](#list-keys)
+  - [Generate Root](#generate-root)
+  - [Generate Intermediate](#generate-intermediate)
+  - [Import CA Certificates and Keys](#import-ca-certificates-and-keys)
+  - [Read Issuer](#read-issuer)
+  - [Update Issuer](#update-issuer)
+  - [Delete Issuer](#delete-issuer)
+  - [Read Key](#read-key)
+  - [Update Key](#update-key)
+  - [Delete Key](#delete-key)
+  - [Delete All Issuers and Keys](#delete-all-issuers-and-keys)
+- [Managing Authority Information](#managing-authority-information)
+  - [List Roles](#list-roles)
+  - [Create/Update Role](#create-update-role)
+  - [Read Role](#read-role)
+  - [Delete Role](#delete-role)
+  - [Read CRL Configuration](#read-crl-configuration)
+  - [Set CRL Configuration](#set-crl-configuration)
+  - [Read URLs](#read-urls)
+  - [Set URLs](#set-urls)
+  - [Read CRL](#read-crl)
+  - [Rotate CRLs](#rotate-crls)
+  - [Tidy](#tidy)
+  - [Tidy Status](#tidy-status)
+- [Notes on Personas](#notes-on-personas)
+- [Cluster Scalability](#cluster-scalability)
 - [Managed Key](#managed-keys) (Enterprise Only)
+- [Vault CLI with DER/PEM responses](#vault-cli-with-der-pem-responses)
 
-## Read CA Certificate
+## Notice About New Multi-Issuer Functionality
 
-This endpoint retrieves the CA certificate _in raw DER-encoded form_. This is a
-bare endpoint that does not return a standard Vault data structure and cannot
-be read by the Vault CLI; use `/pki/cert` for that. If `/pem` is added to the
-endpoint, the CA certificate is returned in PEM format.
+Vault now allows a single PKI mount to have multiple Certificate Authority
+(CA) certificates ("issuers") in a single mount, for the purpose of
+facilitating rotation. All issuers within a single mount are treated as a
+single authority, meaning that Certificate Revocation List (CRL) configuration
+and other authority URLs are common to all issuers and serial numbers will be
+unique across all issuers. However, since each issuer may have a distinct
+subject and keys, different issuers may have different CRLs.
 
-This is an unauthenticated endpoint.
+It is _strongly_ encouraged to limit the scope of CAs within a mount and not
+to mix different types of CAs (roots and intermediates).
 
-| Method | Path            |
-| :----- | :-------------- |
-| `GET`  | `/pki/ca(/pem)` |
+~> **Note**: Some functionality will not work if a default issuer is not
+   configured. Vault automatically selects the default issuer from the
+   current issuing certificate on migration from an older Vault version.
```

Comment:
"older" is going to change over time, could we put down "on migration from a Vault version <=1.10?"